featurex/stimuli/video.py

```diff
@@ -15,7 +16,14 @@ def __init__(self, video, frame_num, filename=None, data=None):
         self.frame_num = frame_num
         self.duration = 1. / video.fps
         self.onset = frame_num * self.duration
-
+    
+class VideoKeyframeStim(VideoFrameStim):
```

Comment:
Apparently "key frame" has a technical meaning in video encoding that's a little different from what we want. Any alternative names?